core/src/Pos/Core/JsonLog/JsonLogT.hs

```diff
@@ -68,41 +65,28 @@ instance MonadBaseControl b m => MonadBaseControl b (JsonLogT m) where
 
     restoreM = restoreM
 
-instance WithLogger m => CanLog (JsonLogT m) where
-
-instance WithLogger m => HasLoggerName (JsonLogT m) where
-
-    askLoggerName = lift askLoggerName
-
-    modifyLoggerName f = hoist (modifyLoggerName f)
-
-
-
-
-
-
 jsonLogDefault
-    :: (ToJSON a, MonadCatch m, MonadIO m, WithLogger m)
+    :: (ToJSON a, MonadIO m, MonadCatch m)
     => JsonLogConfig
-    -> a -> m ()
+    -> a
+    -> m ()
 jsonLogDefault jlc x =
     case jlc of
         JsonLogDisabled -> return ()
         JsonLogConfig v decide -> do
             event <- toEvent <$> timedIO x
             b     <- liftIO (decide event)
-                `catchAny` \e -> do
-                    logWarning $ sformat ("error in deciding whether to json log: "%shown) e
+                `catchAny` \_ -> do
+                    --logWarning logTrace $ sformat ("error in deciding whether to json log: "%shown) e
                     return False
             when b $ liftIO (withMVar v $ \h -> (hPutStrLn h (encode event) >> hFlush h))
-                `catchAny` \e ->
-                    logWarning $ sformat ("can't write json log: "%shown) e
+                `catchAny` \_ -> return ()
+                    --logWarning logTrace $ sformat ("can't write json log: "%shown) e
```

Comment:
Commented out code.

Reply:
JSON logging is being done in another PR. to be submitted this week.